03-types-of-data.Rmd

```diff
@@ -151,7 +151,7 @@ Raster data can come in many different formats.  One of the most common/function
 
 Other file types you will likely encounter include:
 * 1) IMG - A proprietary image format commonly used by ESRI products
-* 2) JPEG2000 - A geospatial version of the common .jpg image type
+* 2) JPEG2000 - A geospatial version of the common .png image type
```

Comment:
This will need to be reverted back from .png to .jpg because the line is used to define JPEG2000 technology